patchview/README.patchview

```diff
@@ -0,0 +1,31 @@
+So what is patchview ? is a wrapper of filterdiff which aim to be used with numbered files.
+
+    $ patchview without args do a: lsdiff --number
+
+    $ patchview with args (for example "-F2-") do a filterdiff with same args.
+
+And we have two scripts for working with git and other two for svn, for svn we got svndiff and svndiffview,
+for git we got gitdiff and gitdiffview .
+
+    $ svndiff or gitdiff (without args) will give the list of files modified
+
+    $ svndiff -F1 or gitdiff -F1 (with args) will show the patch of file #1
+
+    $ svndiffview or gitdiffview (without args) pipe all patch with filerdiff to vim - -R (in ready only mode, easy to
+    quit) , and we will see colored and complete patch,
+
+    $ svndiffview -F2 or gitdiffview -F2 (with args) , it will pipe patch of file #2 to vim - -R
+
+
+and finally we can do scripts in one line, for example difftotrunk.sh script which compare 2 repos (production and devel)
+
+    diff ~/prodution/svn . -ru -x .svn -x local.defs | patchview $*
+
+to view differences of to directories and ./difftotrunk.sh (with args) will give the list of files that I change in devel
+
+    $ ./difftotrunk.sh -F3,4 , will show the patch of file #3 and file #4.
```

Comment:
```suggestion
```

This script isn't included in this PR so let's leave this description for now.

Reply:
I had improve this part and will be the example section